### PR TITLE
Add sdl2 packages entries

### PR DESCRIPTION
**Pull request by f-fl0**
- Add `alpine` entry for `sdl2-image` and `sdl2-mixer`.
- Add `alpine`, `debian` and `ubuntu` entries for `pysdl2` and `pysdl2-dll`.

### CODE REVIEW

File: rosdep/python.yaml [hunk 1]
```diff
@@ -447,6 +447,26 @@ pyros-setup-pip:
...
+pysdl2-dll:
+  alpine:
+    pip:
+      packages: [pysdl2-dll]
+  debian:
+    pip:
+      packages: [pysdl2-dll]
+  ubuntu:
+    pip:
+      packages: [pysdl2-dll]
```
Comment (at-wat) [inline rosdep/python.yaml]: Please add Alpine package to aports-ros-experimental if possible. (Version of the package won't be controlled via pip)

Comment (f-fl0) [reply rosdep/python.yaml]: I added the backports for `pysdl2-dll` and `pysdl2` in https://github.com/seqsense/aports-ros-experimental/pull/713.

Comment (f-fl0) [reply rosdep/python.yaml]: Updated in bc1d41f54400fd2fde26ed6da3b9ca333c135a9f

**Review by at-wat — APPROVED** [review]
LGTM

**Pull request merged**